### DISCUSSION TITLE
Clone URL missing in mobile site

### DISCUSSION THREAD

**Discussion by byron-hawkins** [community/community#17376] [Mobile]
I'm on my mobile trying to clone a wifi driver repo, but somebody at github thought nobody would ever need to sée thé clone URL on a mobile. That's wrong on this case. Please don't hide basic, essential information.

**Comment by brianlovin** [comment]
Hey @byron-hawkins thanks for flagging this — for now, the best thing to do is to tap the `···` in the navigation bar and open the repository profile view in the mobile web browser to grab that link. We'll keep working to make flows like this better integrated in the apps in the future 😊

**Reply by mkowalsky42** [reply]
This doesn't work.

**Reply by jsoref** [reply]
At least for me, I can tap ⋯ and then select `Share`, and then select Copy. 

It's painful, but it generally almost works.

**Reply by Csaba596** [reply]
Csaba596

**Reply by Csaba596** [reply]
Csaba696

**Comment by Mrproex** [comment]
URL is still missing to this day on the mobile and app versions.
I needed to clone a repo and build an app on my phone, why would you hide the url ?

**Comment by luutrutatcavaomaytinh** [comment]
Goole

**Reply by Mrproex** [reply]
We are not on Reddit please refrain from providing dumb answers.

**Comment by darkblaze69** [comment]
I also need this on mobile site. But I am forced to switch to desktop version for such a basic operation...

**Comment by SolsticeSpectrum** [comment]
I installed github as app using Google Chrome. I use termux. I wanted to copy the url. I can't even switch to desktop mode brcause I installed it as app using Google Chrome.

**Comment by emeagenciadigital** [comment]
I can’t work from my phone when I wake up  just because of this missing button 🥲

**Reply by alvarado18** [reply]
That's me right now bro

**Comment by alvarado18** [comment]
Hi

**Reply by owinoalfred** [reply]
This is super stressful!

**Comment by jexek** [comment]
This issue also affects desktop users when the browser window is narrower than ~750px. This can happen easily, for example, when putting two windows side-by-side to copy the clone URL from the browser window to another window.

**Comment by ajn142** [comment]
> This issue also affects desktop users when the browser window is narrower than ~750px. This can happen easily, for example, when putting two windows side-by-side to copy the clone URL from the browser window to another window.

Thanks for this, you pointed me at a workaround. At least for iOS Safari on my 14 Pro, zooming all the way out using the browser’s font size control fixes the issue, and then you can use pinch to zoom and scroll to navigate like you would on a normal desktop-optimized site. I’m curious if the same works on smaller iOS devices or Android devices, don’t have either to test.

![IMG_3966](https://github.com/community/community/assets/5253718/6cf09d9e-41b1-425d-b7a6-33e5e02413b0)

**Comment by TimAmadeoSobania** [comment]
Its been three years... still nothing. Even worse, when I try to do it over the browser, it automatically opens the f****ng app when trying to view a repo.

**Reply by SkyNotion** [reply]
So Annoying
